learning/tour-of-beam/frontend/lib/config/theme/theme.dart

```diff
@@ -78,20 +82,20 @@ AppBarTheme _createAppBarTheme(Color backgroundColor) {
 
 final kLightTheme = ThemeData(
```

Comment:
These may also be non-global.

Reply:
Lint: Avoid defining a class that contains only static members.